README.md

```diff
@@ -42,81 +80,115 @@ Before using the command you have to substitute words in uppercase with proper o
+| prefix   | email   | [CSR Data Loading Pipeline] | Prefix for subject line of the error email.                   |
+| port     | smtp    | 587                         | Port to use for sending emails.                               |
+| username | smtp    |                             | Username for email client, when not needed can be left empty. |
+| password | smtp    |                             | Password for email client.                                    |
+| host     | smtp    | smtp.gmail.com              | Host of the email client.                                     |
+
+
+##Input data
+
+In the drop directory (`drop_dir` in luigi.cfg) the clinical data files should be provided as well as a folder called NGS with all omcs data. 
+Each file in the drop directory has to be accompanied by a sha1 checksum file.
+
+File naming convention:
+- File name: `<filename>.<extension>`
+- Sha1 file: `<filename>.<extension>.sha1`
+
+E.g. data.txt has to have data.txt.sha1 next to it with sha1 hash of the data file. 
+As sha1 hashes 40 characters long the rest of the file gets ignored:
+
+`1625be750dab24057c4c82d62d27298236ebb04c diagnosis.txt`
 
+For more information, see the [CSR data model](https://github.com/thehyve/python_csr2transmart#data-model) description 
+and an example of [input data files](./test_data/dropzone/E2E_TEST_DATA).
+
+## Usage
+
+1. Make sure you have `luigi.cfg` and `email_config.cfg` properly configured (see [configuration section](#configuration)) 
+and the input data is in the proper directory (see [input data section](#input-data)).
+
+2. Install dependencies:
+
+Pipeline requires Python >= 3.6.
+
+``` bash
+$ python -m pip install -r requirements/requirements.txt
 ```
 
-Start luigi daemon, the pipeline workers will communicate with the daemon for work:
+3. Start luigi daemon:
 
 ``` bash
 $ luigid
-
 ```
 
-To start the full pipeline:
+4. Start the full pipeline:
 
 ``` bash
-$ ./run.sh
-
+$ ./scripts/run.sh
 ```
 
-## Fixing manually when something went wrong
 
-When pipeline fails on loading or you want to load data from the history you need to run the data load step only.
+
+### Other available scripts
 
 To load data to transmart only:
 
 ``` bash
-$ ./load_transmart_data.sh
-
+$ ./scripts/load_transmart_data.sh
 ```
 
 To load data to cbioportal only:
 
 ``` bash
-$ ./load_cbioportal_data.sh
-
+$ ./scripts/load_cbioportal_data.sh
 ```
 
 To load data to both systems:
 
 ``` bash
-$ ./load_data.sh
-
+$ ./scripts/load_data.sh
 ```
 
 The pipeline creates files that start with `.done-*`.
 These files created for each successfully finished task of the pipeline.
 To force execution of tasks again you need to remove these files:
 ``` bash
-$ ./remove_done_files.sh
+$ ./scripts/remove_done_files.sh
 ```
 
-That's what happens as part of the script when you run load data task only.
 
-The typical workflow to load version from history might look like this:
+## Test
+
+To run e2e tests:
 ``` bash
-$ cd <data repo>
```

Comment:
git is still used to restore previous data versions, please don't remove this part from the documentation

Reply:
actually this can go in specific user documentation (just informing that Puppet automatically creates a repo for data versioning). leave out here